geode-management/src/main/java/org/apache/geode/management/configuration/ClassName.java

```diff
@@ -42,6 +42,11 @@
   // e.g. alter region --name=regionA --cache-loader=''
   public static ClassName EMPTY = new ClassName("");
 
+  /**
+   * Default constructor used for serialization.
+   */
+  public ClassName() {}
```

Comment:
so, we can add this without adding setters?

Reply:
We have other constructors if they want to specify a class name. And the javadocs on this one states it is for serialization. I would not be against us changing the ClassName impl to be more like our other classes (which user settors) but that would not be part of this pull request.